tests/zkasm/assert.js

```diff
@@ -0,0 +1,68 @@
+const fs = require("fs");
+
+class InstrumentInst {
```

Comment:
`InstrumentInst` is probably copied over from the [helper](https://github.com/near/wasmtime/pull/214/files#diff-e0e611e8f6a5af71b31a8d6ec0c3c888c617d76c39f9ba303ac1c4a8393b788f) in #214. I think here a class name like `Assert` or `AssertHelpers` would make more sense.